examples/flava/utils.py

```diff
@@ -38,4 +38,8 @@ def build_config():
         "max_steps" in config.training.lightning
     ), "lightning config must specify 'max_steps'"
 
+    assert (
```

Comment:
do we need this? lightning works without any strategy override i thought

Reply:
removed